lib/iser.c

```diff
@@ -55,6 +55,8 @@
 
 /* the  name  can  be up to 16 bytes long, including the terminating null byte*/
 #define ISER_CM_THREAD_NAME "iscsi_cm_thread"
+/* let the cmthread shutdown gracefully, wait at most 1000ms */
```

Comment:
Why is this timeout necessary? What would happen if the code below is modified such that it waits indefinitely?